optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/pom.xml

```diff
@@ -38,6 +38,10 @@
       <artifactId>gwtbootstrap3</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.gwtbootstrap3</groupId>
```

Comment:
I can add it if needed, but I've looked how Alexandre was added it to the
datamodeller during BS3 migration and the <inherits
name="org.gwtbootstrap3.extras.select.Select"/>
wasn't added there.

2015-10-01 14:34 GMT+02:00 Michael Anstis <notifications@github.com>:

> In
> optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/pom.xml
> <https://github.com/droolsjbpm/optaplanner-wb/pull/11#discussion_r40907507>
> :
>
> > @@ -38,6 +38,10 @@
> >        <artifactId>gwtbootstrap3</artifactId>
> >      </dependency>
> >      <dependency>
> > +      <groupId>org.gwtbootstrap3</groupId>
>
> If you're adding use of GWT-BS3's Select you also need include an
> additional gwt.xml module.
>
> i.e. <inherits name="org.gwtbootstrap3.extras.select.Select"/>
>
> See https://gwtbootstrap3.github.io/gwtbootstrap3-demo/#select
>
> I couldn't see that in this commit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/droolsjbpm/optaplanner-wb/pull/11/files#r40907507>.
>